packages/ui/src/components/va-accordion/VaAccordion.stories.ts

```diff
@@ -12,8 +12,9 @@ export default {
 
 export const Default = () => ({
   components: { VaAccordion, VaCollapse },
+  data: () => ({ value: [] }),
   template: `
-    <va-accordion>
```

Comment:
I suggest we make accordion stateful by default. (then you can make it false if you need that sweet sweet control)

I'd say - in 90% cases you as user don't need control over collapses.